docs/Encoder.md

```diff
@@ -1,57 +1,103 @@
+
+### 2. DB9 Connector
+
+Each D-sub connecter has 9 pins – 6 for the differential ABZ signals, 2 for the ground signals and 1 for 5V supply to the encoder. A jumper should be installed if the encoder is powered by 5V from the AMDC. There are two jumpers numbered on the silkscreen as JP2 for Encoder 1 and JP3 for Encoder 2, as shown in the block diagram. Encoder 1 and Encoder 2 correspond to the top and bottom encoders in the picture below.
 
-## PCB Layout
 
-All chips and d-sub connectors are located on the top layer. Signal routes from d-sub connectors to differential to single-ended converter and then to level shifter are all located on the top layer as well, while signals from level shifter to PicoZed are located in the intermediate level and each inverting and non-inverting pair of encoder signals are connected between each other by the termination resistor on the bottom layer. All signal traces have 6 mil thickness. Supply and ground traces connected to the chips have thickness of 10 mil.
+A table of the pin mappings for each D-Sub connector is shown below:
 
-It is recommended that bypass capacitors are used on the power supplies to reduce noise, and, if possible, short trace lengths should be used to avoid excessive loading.
+| Pin number | Signal name |
+|------------|--------|
+| 1 | A |
+| 2 | A̅ |
+| 3 | B |
+| 4 | B̅ |
+| 5 | Z |
+| 6 | Z̅ |
+| 7 | 5V |
+| 8 | GND |
+| 9 | GND |
 
+To view the mapping between the AMDC schematic labels, PicoZed pins, and Zynq-7000 balls used in Vivado, take a look at the encoder  section of the [pin mapping document](RevD-PinMapping.md#encoder).
 
+Location of these encoder connectors in the AMDC is shown below:
 
+<img src="images/amdc-encoder-input-highlighted.svg" />
+
+More information on the D-sub connector can be found [here]( https://www.alliedelec.com/m/d/c3366066d9274ddf3c20bc3008518f5b.pdf?src-supplier=Allied+Electronics).
+
+### 3. Differential to Single-Ended Conversion
+
+After passing through the D-sub connector, the differential ABZ signals (A, A̅, A, B̅, Z, Z̅) are converted to the single-ended ABZ signals (OUT A, OUT B, OUT Z) using the [AM26C32](http://www.ti.com/lit/ds/symlink/am26c32.pdf) chip. This chip is supplied by a `Vcc = 5V DC` source from the AMDC. The following table summarizes the absolute maximum ratings taken from the datasheet for this chip:
+
+| Parameter                            | MIN    | MAX           |
+|--------------------------------------|--------|---------------|
+| Supply voltage Vcc                   |        | 7V           |
+| Input voltage (A, A̅, A, B̅, Z or Z̅)   | -11V  | 14V          |
+| Differential input voltage           | -14V  | 14V          |
+| Output voltage (OUT A, OUT B, OUT Z) | -0.5V | (Vcc + 0.5)V |

```

Comment:
Absolute maximum ratings seem unnecessary to provide here. I suggest we remove it and only provide the recommended operating conditions. @npetersen2 and @Anvar-Khamitov, do you agree?

Reply:
I agree as well.